bots/psuteam7bot/prophet.js

```diff
@@ -116,24 +125,16 @@ prophet.takeAttackerAction = (self) => {
     self.log("ATTACKER prophet " + self.id + " taking turn");
 
     //If no base
-    if(self.base == null)
+    if(self.base === null)
     {
         //Set opposite of current coord as target
-        self.potentialEnemyCastleLocation = movement.getAttackerPatrolRoute(self.me, self.map);
-    }
-
-
-    //If no target
-    if(self.potentialEnemyCastleLocation === null)
-    {
-        //Get potential enemy castle locations if Castle didn't send signal
-        self.potentialEnemyCastleLocation = movement.getAttackerPatrolRoute(self.base, self.map);
+        self.base = {x:-1, y:-1};
+        self.target = movement.getMirrorCastle(self.me, self.map);
```

Comment:
This is going to return the opposite corner of the map, correct? I'm not worried about it since it's such an edge case, but theoretically could be issue

Reply:
Mirror location of the bot's spawning/ current location.
if you're concerned about self.base being set to -1 values, it's for a check for one of the signal function (skips listening for signal from base that does not exist)